knowledgerepr/fieldnetwork.py

```diff
@@ -439,6 +442,31 @@ def serialize_network_to_csv(network, path):
             s = str(n) + "," + "node\n"
             f.write(s)
 
+def serialize_network_to_neo4j(network,server="http://neo4j",port=7474,user="neo4j",pwd="aurum"):
```

Comment:
Following up on my previous comment, this function to serialize the network into neo4j could be part of that independent module as well.